spring-cloud-aws-sqs/src/main/java/io/awspring/cloud/sqs/annotation/EnableSqs.java

```diff
@@ -0,0 +1,38 @@
+/*
+ * Copyright 2013-2022 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.awspring.cloud.sqs.annotation;
+
+import io.awspring.cloud.messaging.support.config.MessagingBootstrapConfiguration;
+import io.awspring.cloud.sqs.config.SqsConfigurationSupport;
+import java.lang.annotation.Documented;
+import java.lang.annotation.ElementType;
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import java.lang.annotation.Target;
+import org.springframework.context.annotation.Import;
+
+/**
+ * @author Tomaz Fernandes
+ * @since 3.0
+ */
+@Target(ElementType.TYPE)
+@Retention(RetentionPolicy.RUNTIME)
+@Documented
+@Import({ MessagingBootstrapConfiguration.class, SqsConfigurationSupport.class })
+public @interface EnableSqs {
```

Comment:
From the auto-configuration perspective, yes! We can have something like this in the autoconfigure module. i.e [Kafka](https://github.com/spring-projects/spring-boot/blob/e6f6b2068bcb6dedcedb47b2ba1be2fa92d188aa/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/kafka/KafkaAnnotationDrivenConfiguration.java#L124-L127), [RabbitMQ](https://github.com/spring-projects/spring-boot/blob/0ed9085ba2a2c35a6bdc593b249250c1124929d3/spring-boot-project/spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/amqp/RabbitAnnotationDrivenConfiguration.java#L114-L117)

Reply:
Sure! As a quick note, you can see in both examples how the `xxxConfigUtils` class is used - that's supposed to be the same role as the `MessagingConfigUtils` class I created here.